Intro_to_Recommendation_Systems/Intro_Recommender.ipynb

```diff
@@ -83,6 +83,7 @@
     "print(\"GPU: \", get_gpu_name())\n",
     "print(\"GPU memory: \", get_gpu_memory())\n",
     "print(\"CUDA: \", get_cuda_version())\n",
+    "print(torch.cuda.get_device_name())", #hopefully shows gpu name
```

Comment:
This breaks the notebook, because you are modifying directly the raw ipynb file. To make this work, you would need to open the notebook with the jupyter server and add the line there